cms/static/js/views/metadata.js

```diff
@@ -220,6 +225,44 @@ function(BaseView, _, MetadataModel, AbstractEditor, FileUpload, UploadDialog, V
 
     });
 
+    Metadata.NumberWithWarning = Metadata.Number.extend({
+        initialize: function() {
+            Metadata.Number.prototype.initialize.apply(this);
+            var options = this.model.get("options");
+            this.msg = options["warning"];
+            // show initial warning if necessary
```

Comment:
Does this comment add context that would be otherwise unavailable without it? Or could we remove it?